### PR TITLE
Recreated the HVAC XML files based on a new Excel sheet provided by D…

### PR DESCRIPTION
**Pull request by wvreeven**
…atControl.

### CODE REVIEW

**Review by r-owen — APPROVED** [review]
Looks good. Two small suggestions to consider (one trivial).

File: sal_interfaces/HVAC/HVAC_Commands.xml [hunk 1]
```diff
@@ -1,10 +1,719 @@
...
-<?xml-stylesheet type='text/xsl' href='http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsl'?>
+<?xml version="1.0" encoding="utf-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsl"?>
 <SALCommandSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsd">
   <SALCommand>
     <Subsystem>HVAC</Subsystem>
-    <EFDB_Topic>HVAC_command_vea01P01</EFDB_Topic>
-    <Alias>vea01P01</Alias>
+    <EFDB_Topic>HVAC_command_chiller01P01_config</EFDB_Topic>
+    <Alias>chiller01P01_config</Alias>
```
[the remark this answers is not in the document]
Comment (wvreeven) [reply sal_interfaces/HVAC/HVAC_Commands.xml]: Good suggestion to remove the Alias tag. I have done that now.

As for the Description field, this is an ongoing discussion within our team. I prefer to keep the rough English translations for now, even if they appear to be useless. I like your suggestion to keep a file of command descriptions and mesh that with the data from the vendor. I created DM-27642 for that.

[the remark this answers is not in the document]
Comment (r-owen) [reply sal_interfaces/HVAC/HVAC_Commands.xml]: Just to be sure I was clear: I am suggesting a Description tag for each command; right now you only have one per field, but not one per command.

[the remark this answers is not in the document]
Comment (wvreeven) [reply sal_interfaces/HVAC/HVAC_Commands.xml]: Thanks for that clarification. I added a note to the Jira ticket.

**Pull request merged**